app/lib/actions/katello/applicability/host_generate.rb

```diff
@@ -0,0 +1,32 @@
+module Actions
+  module Katello
+    module Applicability
+      module Host
+        class Generate < Actions::EntryAction
+          # This should be run through Katello::Events::GenerateHostApplicability
+
+          input_format do
+            param :host_id, Integer
+          end
+
+          def run
+            content_facet = ::Host.find(input[:host_id]).content_facet
+
+            package_content_ids = ::Katello::Applicability::ApplicableContentHelper.new(::Katello::Rpm, content_facet).calculate_and_import
+            erratum_content_ids = ::Katello::Applicability::ApplicableContentHelper.new(::Katello::Erratum, content_facet).calculate_and_import
+
+            output[:response] = "package_content_ids were #{package_content_ids}. erratum_content_ids were #{erratum_content_ids}."
```

Comment:
we probably don't need this in the task response, we could include a count?

Reply:
Yeah, that was for me during debugging